spec/defines/timer_spec.rb

```diff
@@ -29,7 +29,7 @@
           it {
             expect(subject).to contain_systemd__unit_file('foobar.service').with(
               content: "[Service]\nExecStart=/bin/touch /tmp/foobar"
-            )
+            ).that_comes_before('Systemd::Unit_File[\'foobar.timer\']')
```

Comment:
No need to quote this. I if you apply this everywhere the tests will pass
```suggestion
            ).that_comes_before('Systemd::Unit_File[foobar.timer]')
```

Reply:
Many thanks for pointing this out! That was unexpected, I thought the syntax would be similar to that used in Puppet directly. 
Fixed and force-pushed :-).

Reply:
It seems tests still did not like this, I changed:
`Systemd::Unit_File` => `Systemd::Unit_file` now, assuming capitalization needs to match explicitly. Let's see if that makes things more green.

Reply:
That did the trick :smile:.